src/opnsense/service/templates/OPNsense/Proxy/squid.conf

```diff
@@ -444,6 +444,10 @@ visible_hostname {{OPNsense.proxy.general.VisibleHostname}}
 # Define visible email
 cache_mgr {{OPNsense.proxy.general.VisibleEmail}}
 {% endif %}
+{% if helpers.exists('OPNsense.proxy.general.connecttimeout') and OPNsense.proxy.general.connecttimeout != '' %}
```

Comment:
`{% if not helpers.empty('OPNsense.proxy.general.connecttimeout') %}` <--

Reply:
Didn't know this one, thx :)